assets/helpers/user/user.js

```diff
@@ -27,7 +27,14 @@ const init = (dispatch: Function) => {
 
   const uatMode = window.guardian && window.guardian.uatMode;
 
-  const emailFromSession: ?string = getSession('gu.email');
+  function getEmailFromPage(): ?string {
```

Comment:
This function isn't getEmailFromPage, it is getEmailFromPageAndFallBackToCookie

Maybe there is something more concise than that

Reply:
How about  generic `getEmailFromBrowser`?

Reply:
Fine with me